Database/customerdb.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs')
+if(fs.existsSync('.env')){
+  require('dotenv').config()
+}
+
+const connectingString = process.env.DATABASE_URL
+
+const pgp = require('pg-promise')()
+
+const db = pgp(connectingString)
```

Comment:
I would group the above three `const` declarations together, at the top of the module, to provide a visual clue that they are related, and are for setup and configuration.